test/functional/assertion-helper.js

```diff
@@ -66,38 +71,49 @@ exports.errorInEachBrowserNotContains = function errorInEachBrowserNotContains (
     }
 };
 
-exports.checkScreenshotsCreated = function checkScreenshotsCreated (withoutScreenshot, count, customPath) {
-    var screenshotsPath     = '___test-screenshots___';
-    var expectedCount       = count || 2;
-    var screenshotDirExists = isDirExists(screenshotsPath);
+exports.checkScreenshotsAreNotCreated = function () {
```

Comment:
Hm, just saw this. It's two different functions I suppose. 
The first checks screenshots. The second clears screenshots dir. And I think we should call the second after each test